util/src/main/java/org/hyperledger/besu/util/Subscribers.java

```diff
@@ -152,4 +152,14 @@ public int getSubscriberCount() {
       return 0;
     }
   }
+
+  public interface Unsubscriber {
```

Comment:
This creates two patterns now, both of which are present in the codebase, a long subscriber ID and a unsubscriber object.  We should pick one.  Perhaps for this release we stick with the existing pattern and in another PR do a full switchover if it is better.  Having multiple subscribe patterns (that are already different from the Java event listener pattern) makes the code less consistant and readable.

Reply:
Good points, I can cut this.  I had a more complicated set-up during development that made tracking subscribers by ids complex, which is why I originally introduced this.  But there's no need for it at this point.

Reply:
Removed